compiler/testSuite/semtype/proj2-t .bal

```diff
@@ -0,0 +1,5 @@
+type I int;
+
+// @type A[0] = I
+// @type A[1] = I
```

Comment:
We want to test `A[int]` too.

Reply:
can we test this with `listProj` since we need  the key value for listProj?

Reply:
Oh yes, you're right. We can't do that.

Reply:
We can only test that with listMemberType which will give us `int?`

Reply:
We could add 
```
type I int;
type I2 int?;

// @type A[I] = I2
```
just for testing purposes

Reply:
Yes.